src/components/metadataEditor/metadataEditor.js

```diff
@@ -298,6 +298,42 @@ function bindAll(elems, eventName, fn) {
     }
 }
 
+function onResetClick(e) {
+    const resetElementId = ['#txtName', '#txtOriginalName', '#txtSortName', '#txtCommunityRating', '#txtCriticRating', '#txtIndexNumber',
+        '#txtAirsBeforeSeason', '#txtAirsAfterSeason', '#txtAirsBeforeEpisode', '#txtParentIndexNumber', '#txtAlbum',
+        '#txtAlbumArtist', '#txtArtist', '#txtOverview', '#selectStatus', '#txtAirTime', '#txtPremiereDate', '#txtDateAdded', '#txtEndDate',
+        '#txtProductionYear', '#selectHeight', '#txtOriginalAspectRatio', '#select3dFormat', '#selectOfficialRating', '#selectCustomRating',
+        '#txtSeriesRuntime', '#txtTagline'];
+    const form = e.target.parentElement.parentElement.parentElement;
```

Comment:
Untested but I think we can do this instead of chaining `parentElement`.

```suggestion
    const form = currentContext?.querySelector('form');
```